### ISSUE TITLE
2025-06-04, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#190]
# Today's Arxiv Papers

(1) [Entity-Augmented Neuroscience Knowledge Retrieval Using Ontology and Semantic Understanding Capability of LLM](http://arxiv.org/abs/2506.03145v1), 2025-06-03

(2) [Not All Tokens Are Meant to Be Forgotten](http://arxiv.org/abs/2506.03142v1), 2025-06-03

(3) [SVGenius: Benchmarking LLMs in SVG Understanding, Editing and Generation](http://arxiv.org/abs/2506.03139v1), 2025-06-03

(4) [Co-Evolving LLM Coder and Unit Tester via Reinforcement Learning](http://arxiv.org/abs/2506.03136v1), 2025-06-03

(5) [Native-Resolution Image Synthesis](http://arxiv.org/abs/2506.03131v1), 2025-06-03

(6) [AnimeShooter: A Multi-Shot Animation Dataset for Reference-Guided Video Generation](http://arxiv.org/abs/2506.03126v1), 2025-06-03

(7) [AUTOCIRCUIT-RL: Reinforcement Learning-Driven LLM for Automated Circuit Topology Generation](http://arxiv.org/abs/2506.03122v1), 2025-06-03

(8) [Critique-GRPO: Advancing LLM Reasoning with Natural Language and Numerical Feedback](http://arxiv.org/abs/2506.03106v1), 2025-06-03

(9) [Retrieval-Augmented Generation as Noisy In-Context Learning: A Unified Theory and Risk Bounds](http://arxiv.org/abs/2506.03100v1), 2025-06-03

(10) [TalkingMachines: Real-Time Audio-Driven FaceTime-Style Video via Autoregressive Diffusion Models](http://arxiv.org/abs/2506.03099v1), 2025-06-03

(11) [DPO Learning with LLMs-Judge Signal for Computer Use Agents](http://arxiv.org/abs/2506.03095v1), 2025-06-03

(12) [Literary Evidence Retrieval via Long-Context Language Models](http://arxiv.org/abs/2506.03090v1), 2025-06-03

(13) [StreamBP: Memory-Efficient Exact Backpropagation for Long Sequence Training of LLMs](http://arxiv.org/abs/2506.03077v1), 2025-06-03

(14) [EDITOR: Effective and Interpretable Prompt Inversion for Text-to-Image Diffusion Models](http://arxiv.org/abs/2506.03067v1), 2025-06-03

(15) [MAEBE: Multi-Agent Emergent Behavior Framework](http://arxiv.org/abs/2506.03053v1), 2025-06-03

(16) [Facts Do Care About Your Language: Assessing Answer Quality of Multilingual LLMs](http://arxiv.org/abs/2506.03051v1), 2025-06-03

(17) [Towards Analyzing and Understanding the Limitations of VAPO: A Theoretical Perspective](http://arxiv.org/abs/2506.03038v1), 2025-06-03

(18) [Leveraging Information Retrieval to Enhance Spoken Language Understanding Prompts in Few-Shot Learning](http://arxiv.org/abs/2506.03035v1), 2025-06-03

(19) [TestAgent: An Adaptive and Intelligent Expert for Human Assessment](http://arxiv.org/abs/2506.03032v1), 2025-06-03

(20) [GenFair: Systematic Test Generation for Fairness Fault Detection in Large Language Models](http://arxiv.org/abs/2506.03024v1), 2025-06-03

(21) [Conditioning Large Language Models on Legal Systems? Detecting Punishable Hate Speech](http://arxiv.org/abs/2506.03009v1), 2025-06-03

(22) [A Preference-Driven Methodology for High-Quality Solidity Code Generation](http://arxiv.org/abs/2506.03006v1), 2025-06-03

(23) [Linear Spatial World Models Emerge in Large Language Models](http://arxiv.org/abs/2506.02996v1), 2025-06-03

(24) [It's Not a Walk in the Park! Challenges of Idiom Translation in Speech-to-text Systems](http://arxiv.org/abs/2506.02995v1), 2025-06-03

(25) [Mitigating Manipulation and Enhancing Persuasion: A Reflective Multi-Agent Approach for Legal Argument Generation](http://arxiv.org/abs/2506.02992v1), 2025-06-03

(26) [Performance of leading large language models in May 2025 in Membership of the Royal College of General Practitioners-style examination questions: a cross-sectional analysis](http://arxiv.org/abs/2506.02987v1), 2025-06-03

(27) [Expanding before Inferring: Enhancing Factuality in Large Language Models through Premature Layers Interpolation](http://arxiv.org/abs/2506.02973v1), 2025-06-03

(28) [Memory-Efficient and Privacy-Preserving Collaborative Training for Mixture-of-Experts LLMs](http://arxiv.org/abs/2506.02965v1), 2025-06-03

(29) [FlowerTune: A Cross-Domain Benchmark for Federated Fine-Tuning of Large Language Models](http://arxiv.org/abs/2506.02961v1), 2025-06-03

(30) [HACo-Det: A Study Towards Fine-Grained Machine-Generated Text Detection under Human-AI Coauthoring](http://arxiv.org/abs/2506.02959v1), 2025-06-03

(31) [Towards More Effective Fault Detection in LLM-Based Unit Test Generation](http://arxiv.org/abs/2506.02954v1), 2025-06-03

(32) [Adaptive Graph Pruning for Multi-Agent Communication](http://arxiv.org/abs/2506.02951v1), 2025-06-03

(33) [Quantitative LLM Judges](http://arxiv.org/abs/2506.02945v1), 2025-06-03

(34) [A Multi-agent LLM-based JUit Test Generation with Strong Oracles](http://arxiv.org/abs/2506.02943v1), 2025-06-03

(35) [Memory-Efficient Split Federated Learning for LLM Fine-Tuning on Heterogeneous Mobile Devices](http://arxiv.org/abs/2506.02940v1), 2025-06-03

(36) [QKV Projections Require a Fraction of Their Memory](http://arxiv.org/abs/2506.02939v1), 2025-06-03

(37) [Large Processor Chip Model](http://arxiv.org/abs/2506.02929v1), 2025-06-03

(38) [INESC-ID @ eRisk 2025: Exploring Fine-Tuned, Similarity-Based, and Prompt-Based Approaches to Depression Symptom Identification](http://arxiv.org/abs/2506.02924v1), 2025-06-03

(39) [Sample, Predict, then Proceed: Self-Verification Sampling for Tool Use of LLMs](http://arxiv.org/abs/2506.02918v1), 2025-06-03

(40) [Cell-o1: Training LLMs to Solve Single-Cell Reasoning Puzzles with Reinforcement Learning](http://arxiv.org/abs/2506.02911v1), 2025-06-03

(41) [Scaling Fine-Grained MoE Beyond 50B Parameters: Empirical Evaluation and Practical Insights](http://arxiv.org/abs/2506.02890v1), 2025-06-03

(42) [CoT is Not True Reasoning, It Is Just a Tight Constraint to Imitate: A Theory Perspective](http://arxiv.org/abs/2506.02878v1), 2025-06-03

(43) [It's the Thought that Counts: Evaluating the Attempts of Frontier LLMs to Persuade on Harmful Topics](http://arxiv.org/abs/2506.02873v1), 2025-06-03

(44) [Pan-Arctic Permafrost Landform and Human-built Infrastructure Feature Detection with Vision Transformers and Location Embeddings](http://arxiv.org/abs/2506.02868v1), 2025-06-03

(45) [Demystifying Reasoning Dynamics with Mutual Information: Thinking Tokens are Information Peaks in LLM Reasoning](http://arxiv.org/abs/2506.02867v1), 2025-06-03

(46) [BNPO: Beta Normalization Policy Optimization](http://arxiv.org/abs/2506.02864v1), 2025-06-03

(47) [Tru-POMDP: Task Planning Under Uncertainty via Tree of Hypotheses and Open-Ended POMDPs](http://arxiv.org/abs/2506.02860v1), 2025-06-03

(48) [ATAG: AI-Agent Application Threat Assessment with Attack Graphs](http://arxiv.org/abs/2506.02859v1), 2025-06-03

(49) [METok: Multi-Stage Event-based Token Compression for Efficient Long Video Understanding](http://arxiv.org/abs/2506.02850v1), 2025-06-03

(50) [CLONE: Customizing LLMs for Efficient Latency-Aware Inference at the Edge](http://arxiv.org/abs/2506.02847v1), 2025-06-03

(51) [TaxAgent: How Large Language Model Designs Fiscal Policy](http://arxiv.org/abs/2506.02838v1), 2025-06-03

(52) [TO-GATE: Clarifying Questions and Summarizing Responses with Trajectory Optimization for Eliciting Human Preference](http://arxiv.org/abs/2506.02827v1), 2025-06-03

(53) [ProcrustesGPT: Compressing LLMs with Structured Matrices and Orthogonal Transformations](http://arxiv.org/abs/2506.02818v1), 2025-06-03

(54) [Rethinking the effects of data contamination in Code Intelligence](http://arxiv.org/abs/2506.02791v1), 2025-06-03

(55) [Rethinking Dynamic Networks and Heterogeneous Computing with Automatic Parallelization](http://arxiv.org/abs/2506.02787v1), 2025-06-03

(56) [Reuse or Generate? Accelerating Code Editing via Edit-Oriented Speculative Decoding](http://arxiv.org/abs/2506.02780v1), 2025-06-03

(57) [Exploiting the English Vocabulary Profile for L2 word-level vocabulary assessment with LLMs](http://arxiv.org/abs/2506.02758v1), 2025-06-03

(58) [Prompt-Unseen-Emotion: Zero-shot Expressive Speech Synthesis with Prompt-LLM Contextual Knowledge for Mixed Emotions](http://arxiv.org/abs/2506.02742v1), 2025-06-03

(59) [Why do AI agents communicate in human language?](http://arxiv.org/abs/2506.02739v1), 2025-06-03

(60) [RACE-Align: Retrieval-Augmented and Chain-of-Thought Enhanced Preference Alignment for Large Language Models](http://arxiv.org/abs/2506.02726v1), 2025-06-03

(61) [Benchmarking and Advancing Large Language Models for Local Life Services](http://arxiv.org/abs/2506.02720v1), 2025-06-03

(62) [Heterogeneous Group-Based Reinforcement Learning for LLM-based Multi-Agent Systems](http://arxiv.org/abs/2506.02718v1), 2025-06-03

(63) [Open-Set Living Need Prediction with Large Language Models](http://arxiv.org/abs/2506.02713v1), 2025-06-03

(64) [Shaking to Reveal: Perturbation-Based Detection of LLM Hallucinations](http://arxiv.org/abs/2506.02696v1), 2025-06-03

(65) [MASTER: Enhancing Large Language Model via Multi-Agent Simulated Teaching](http://arxiv.org/abs/2506.02689v1), 2025-06-03

(66) [Decompose, Plan in Parallel, and Merge: A Novel Paradigm for Large Language Models based Planning with Multiple Constraints](http://arxiv.org/abs/2506.02683v1), 2025-06-03

(67) [TL;DR: Too Long, Do Re-weighting for Effcient LLM Reasoning Compression](http://arxiv.org/abs/2506.02678v1), 2025-06-03

(68) [EvaLearn: Quantifying the Learning Capability and Efficiency of LLMs via Sequential Problem Solving](http://arxiv.org/abs/2506.02672v1), 2025-06-03

(69) [Are Economists Always More Introverted? Analyzing Consistency in Persona-Assigned LLMs](http://arxiv.org/abs/2506.02659v1), 2025-06-03

(70) [Computational Thinking Reasoning in Large Language Models](http://arxiv.org/abs/2506.02658v1), 2025-06-03

(71) [From Prompts to Protection: Large Language Model-Enabled In-Context Learning for Smart Public Safety UAV](http://arxiv.org/abs/2506.02649v1), 2025-06-03

(72) [Truly Assessing Fluid Intelligence of Large Language Models through Dynamic Reasoning Evaluation](http://arxiv.org/abs/2506.02648v1), 2025-06-03

(73) [KVCache Cache in the Wild: Characterizing and Optimizing KVCache Cache at a Large Cloud Provider](http://arxiv.org/abs/2506.02634v1), 2025-06-03

(74) [HAM: A Hyperbolic Step to Regulate Implicit Bias](http://arxiv.org/abs/2506.02630v1), 2025-06-03

(75) [EssayBench: Evaluating Large Language Models in Multi-Genre Chinese Essay Writing](http://arxiv.org/abs/2506.02596v1), 2025-06-03

(76) [EALG: Evolutionary Adversarial Generation of Language Model-Guided Generators for Combinatorial Optimization](http://arxiv.org/abs/2506.02594v1), 2025-06-03

(77) [Beyond the Surface: Measuring Self-Preference in LLM Judgments](http://arxiv.org/abs/2506.02592v1), 2025-06-03

(78) [On Generalization across Measurement Systems: LLMs Entail More Test-Time Compute for Underrepresented Cultures](http://arxiv.org/abs/2506.02591v1), 2025-06-03

(79) [Evaluating Named Entity Recognition Models for Russian Cultural News Texts: From BERT to LLM](http://arxiv.org/abs/2506.02589v1), 2025-06-03

(80) [IndoSafety: Culturally Grounded Safety for LLMs in Indonesian Languages](http://arxiv.org/abs/2506.02573v1), 2025-06-03

(81) [HATA: Trainable and Hardware-Efficient Hash-Aware Top-k Attention for Scalable Large Model Inference](http://arxiv.org/abs/2506.02572v1), 2025-06-03

(82) [MLaGA: Multimodal Large Language and Graph Assistant](http://arxiv.org/abs/2506.02568v1), 2025-06-03

(83) [Pruning General Large Language Models into Customized Expert Models](http://arxiv.org/abs/2506.02561v1), 2025-06-03

(84) [Kernel-based Unsupervised Embedding Alignment for Enhanced Visual Representation in Vision-language Models](http://arxiv.org/abs/2506.02557v1), 2025-06-03

(85) [Response-Level Rewards Are All You Need for Online Reinforcement Learning in LLMs: A Mathematical Perspective](http://arxiv.org/abs/2506.02553v1), 2025-06-03

(86) [Technical Report for Ego4D Long-Term Action Anticipation Challenge 2025](http://arxiv.org/abs/2506.02550v1), 2025-06-03

(87) [CyberGym: Evaluating AI Agents' Cybersecurity Capabilities with Real-World Vulnerabilities at Scale](http://arxiv.org/abs/2506.02548v1), 2025-06-03

(88) [Attention Knows Whom to Trust: Attention-based Trust Management for LLM Multi-Agent Systems](http://arxiv.org/abs/2506.02546v1), 2025-06-03

(89) [CoRe-MMRAG: Cross-Source Knowledge Reconciliation for Multimodal RAG](http://arxiv.org/abs/2506.02544v1), 2025-06-03

(90) [VisuRiddles: Fine-grained Perception is a Primary Bottleneck for Multimodal Large Language Models in Abstract Visual Reasoning](http://arxiv.org/abs/2506.02537v1), 2025-06-03

(91) [Answer Convergence as a Signal for Early Stopping in Reasoning](http://arxiv.org/abs/2506.02536v1), 2025-06-03

(92) [Automated Web Application Testing: End-to-End Test Case Generation with Large Language Models and Screen Transition Graphs](http://arxiv.org/abs/2506.02529v1), 2025-06-03

(93) [RelationAdapter: Learning and Transferring Visual Relation with Diffusion Transformers](http://arxiv.org/abs/2506.02528v1), 2025-06-03

(94) [Hardware-Centric Analysis of DeepSeek's Multi-Head Latent Attention](http://arxiv.org/abs/2506.02523v1), 2025-06-03

(95) [Think Twice, Act Once: A Co-Evolution Framework of LLM and RL for Large-Scale Decision Making](http://arxiv.org/abs/2506.02522v1), 2025-06-03

(96) [Learning Together to Perform Better: Teaching Small-Scale LLMs to Collaborate via Preferential Rationale Tuning](http://arxiv.org/abs/2506.02519v1), 2025-06-03

(97) [FinChain: A Symbolic Benchmark for Verifiable Chain-of-Thought Financial Reasoning](http://arxiv.org/abs/2506.02515v1), 2025-06-03

(98) [To Embody or Not: The Effect Of Embodiment On User Perception Of LLM-based Conversational Agents](http://arxiv.org/abs/2506.02514v1), 2025-06-03

(99) [M$^3$FinMeeting: A Multilingual, Multi-Sector, and Multi-Task Financial Meeting Understanding Evaluation Dataset](http://arxiv.org/abs/2506.02510v1), 2025-06-03

(100) [In-context Clustering-based Entity Resolution with Large Language Models: A Design Space Exploration](http://arxiv.org/abs/2506.02509v1), 2025-06-03
